links/src/utils/__tests__/generateLinks.test.ts

```diff
@@ -1,11 +1,56 @@
-import { generateLinks } from "../generateLinks";
+import { generateLinks, totree } from "../generateLinks";
 import { test } from "uvu";
 import * as assert from "uvu/assert";
 
-test("generateLinks:", () => {
+type types = {
+	id: number;
+	parent_id: number;
+};
```

Comment:
```suggestion
```

It's better to define your types next to your logic under `links/src/utils/generateLinks.ts`